src/Symfony/Component/Process/Process.php

```diff
@@ -1210,9 +1210,13 @@ public function setInput($input)
      * @param bool $inheritEnv
      *
      * @return self The current Process instance
+     *
+     * @deprecated since Symfony 4.4, env variables are always inherited
      */
     public function inheritEnvironmentVariables($inheritEnv = true)
     {
+        @trigger_error(sprintf('The "%s()" method is deprecated since Symfony 4.4 and will be removed in 5.0. Env variables are always inherited.', __METHOD__), E_USER_DEPRECATED);
```

Comment:
don't we get rid of "and will be removed in 5.0"?

Reply:
Right! Removed.